src/pages/public_registration/action_accordions/RefreshPublishingRequestButton.tsx

```diff
@@ -0,0 +1,30 @@
+import RefreshIcon from '@mui/icons-material/Refresh';
+import { ButtonProps } from '@mui/lab';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useTranslation } from 'react-i18next';
+import { Button } from '@mui/material';
```

Comment:
Noen grunn til at ButtonProps er importert fra lab i stedet for material?

```suggestion
import { dataTestId } from '../../../utils/dataTestIds';
import { useTranslation } from 'react-i18next';
import { Button, ButtonProps } from '@mui/material';
```